__tests__/tests/components/ImageZoom.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { ImageZoom } from 'components';
+
+describe('<ImageZoom />', () => {
+  const defaultProps = {
+    style: { backgroundColor: 'red' },
+    uri: { uri: 'dummy.png' },
+  };
+
+  it('should render clickable image', () => {
+    const wrapper = shallow(<ImageZoom {...defaultProps} />);
+
+    const clickableImg = wrapper.find({ nativeId: 'image-zoom-clickable-img' });
+
+    expect(clickableImg.length).toBe(1);
+  });
+
+  it('should render modal when the user presses Touchable', () => {
+    const wrapper = shallow(<ImageZoom {...defaultProps} />);
+
+    wrapper.simulate('press');
+
+    const modal = wrapper.find({ nativeId: 'image-zoom-modal' });
+
+    expect(modal.length).toBe(1);
+  });
+
+  it('should close modal when onRequestClose is called', () => {
+    const wrapper = shallow(<ImageZoom {...defaultProps} />);
+    wrapper.setState({ imgZoom: true });
+
+    wrapper.props().onRequestClose();
```

Comment:
I get this test and it's ok, but it will be better if you check that the modal close after simulating a press (for example) on X or Y component. By doing this, you'll ensure that a real event did close the modal.